lib/index.js

```diff
@@ -41,6 +41,22 @@ bedrock.events.on('bedrock-mongodb.ready', function init(callback) {
 });
 
 /**
+ * Check for the existence of an identity.
+ *
+ * @param id the ID of the identity to check.
+ * @param callback(err, exists) called once the operation completes.
+ */
+api.exists = function(id, callback) {
+  database.collections.identity.find({
```

Comment:
@dlongley I'm referring to the second performance check that includes the implementation with a permission check.  We take a ~5% hit by adding the permission check even when used with a null actor.